tap_mssql/sync_strategies/full_table.py

```diff
@@ -2,9 +2,18 @@
 # pylint: disable=duplicate-code,too-many-locals,simplifiable-if-expression
```

Comment:
Can we add a Fast Sync section to the readme.md? There are a few required config changes like - "fastsync_batch_rows" that should be called out.

Reply:
updated